cognite/client/utils/_version_checker.py

```diff
@@ -9,17 +9,19 @@ def check_if_version_exists(package_name: str, version: str):
     return version in versions
 
 
-def get_newest_version_in_major_release(package_name: str, version: str):
+def get_newest_version_in_major_release(package_name: str, version: str, verify_ssl: bool):
     major, minor, micro, pr_cycle, pr_version = _parse_version(version)
-    versions = get_all_versions(package_name)
+    versions = get_all_versions(package_name, verify_ssl)
     for v in versions:
         if _is_newer_major(v, version):
             major, minor, micro, pr_cycle, pr_version = _parse_version(v)
     return _format_version(major, minor, micro, pr_cycle, pr_version)
 
 
-def get_all_versions(package_name: str):
-    res = requests.get("https://pypi.python.org/simple/{}/#history".format(package_name))
+def get_all_versions(package_name: str, verify_ssl: bool):
+    session = requests.Session()
```

Comment:
I don't configure the session with HTTPAdapter as in _api_client.py. Is this necessary?

Reply:
I think you can skip the session and just pass the verify flag directly to `requests.get`. Either that or use an existing session - no need to create a session just for this one request.

Reply:
Agreed, I'll change this.